web/src/test/scala/net/wiringbits/forms/UpdateInfoFormDataSpec.scala

```diff
@@ -0,0 +1,56 @@
+package net.wiringbits.forms
+
+import net.wiringbits.common.models.{Email, Name}
+import org.scalatest.matchers.must.Matchers._
+import org.scalatest.wordspec.AnyWordSpec
+
+class UpdateInfoFormDataSpec extends AnyWordSpec {
```

Comment:
Thanks for following the pattern and writing tests for forms, even if they are pretty simple.